docs/serving/setting-up-a-logging-plugin.md

```diff
@@ -1,90 +0,0 @@
----
-title: "Setting up a logging plugin"
```

Comment:
Reading this, this is all about the integrated monitoring bundle, not general fluentd advice.

Reply:
So, it can go or not? :slightly_smiling_face:

Reply:
It can go, sorry.